### PR TITLE
Readme upadated

### PR DESCRIPTION
**Pull request by chetnaakhandagale**
Please check the changes I made. This is my first time in open source contribution. Suggest anything if needed.

## Issue that this pull request solves

Closes: #140 

## Proposed changes

I have made very few changes in the readme. 

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ x] Other (please describe): Only few punctuations were changed and some grammatical mistakes are corrected.

## Checklist

_Put an `x` in the boxes that apply_

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings

## Screenshots

Please attach the screenshots of the changes made in case of change in the user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: ReadME.md [hunk 1]
```diff
@@ -143,7 +143,7 @@ This project is part of the [GirlScript Summer of Code 2022](https://gssoc.girls
 | **[Praveen Kumar Purushothaman](https://praveen.science/)**  |
 
 >**Need help?**
-Feel free to contact me at [catsintech@praveen.science](mailto:catsintech@praveen.science).
+Feel free to contact me at [catsintech@praveen.science](mail to:catsintech@praveen.science).
```
Comment (code-reaper08) [inline ReadME.md:146]: Hello @chetnaakhandagale 👋 
This should be retained as original to work as expected!

File: ReadME.md [hunk 2]
```diff
@@ -110,15 +110,15 @@ This project is part of the [GirlScript Summer of Code 2022](https://gssoc.girls
 
 ### Frequently Asked Questions(FAQs) for [GSSoC '22](https://gssoc.girlscript.tech/)
 
-> This project is accepting contributions under [GirlScript Summer of Code 2022](https://gssoc.girlscript.tech/) - three months long Open Source program conducted by [GirlScript Foundation](https://gssoc.girlscript.tech/).
+> This project is accepting contributions under [GirlScript Summer of Code 2022](https://gssoc.girlscript.tech/) - three monthly long Open Source program conducted by [GirlScript Foundation](https://gssoc.girlscript.tech/).
```
Comment (code-reaper08) [inline ReadME.md:113]: Guess it's **three-month** :-)

**Review by code-reaper08 — CHANGES_REQUESTED** [review]
I've left a review 👍

File: ReadME.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 [![first-timers-only](https://img.shields.io/badge/first--timers--only-friendly-tomato.svg?style=flat&logo=git)](https://github.com/CatsInTech/Rezume/issues) [![GitHub stars](https://img.shields.io/github/stars/CatsInTech/Rezume.svg?logo=github)](https://github.com/CatsInTech/Rezume/stargazers) [![GitHub forks](https://img.shields.io/github/forks/CatsInTech/Rezume.svg?logo=github&color=teal)](https://github.com/CatsInTech/Rezume/network) [![GitHub top language](https://img.shields.io/github/languages/top/CatsInTech/Rezume?color=yellow&logo=javascript)](https://github.com/CatsInTech/Rezume) 
 
-Rezume is a free online portal for generating resumes without worrying about the styling and formatting. In the web portal, you only need to input the basic information which you want to have on your resume and it will automatically generate a decent resume for you.
+Rezume is a free online portal for generating resumes without worrying about styling and formatting. In the web portal, you only need to input the basic information which you want to have on your resume and it will automatically generate a decent resume for you.
```
Comment (praveenscience) [inline ReadME.md:5]: Not needed change.

File: ReadME.md [hunk 4]
```diff
@@ -36,9 +36,9 @@ Rezume is a free online portal for generating resumes without worrying about the
 As this is a React JS application, you need the following to support your development environment.
 - [ ] [Git](https://git-scm.com/downloads)
 - [ ] [Node.js](https://nodejs.org/en/) (Please go with any latest LTS versions)
-- [ ] [Linux Based Terminal](https://gitforwindows.org/), if you're using Windows and installed with Git.
+- [ ] [Linux Based Terminal](https://gitforwindows.org/) if you're using Windows and installed with Git.
```
Comment (praveenscience) [inline ReadME.md:39]: Need a comma - basic grammar.

File: ReadME.md [hunk 5]
```diff
@@ -36,9 +36,9 @@ Rezume is a free online portal for generating resumes without worrying about the
...
 - [ ] [Google Chrome](https://www.google.com/chrome/) or a decent latest web browser.
-- [ ] **[Visual Studio Code](https://code.visualstudio.com/)** is the go-to editor for some complex languages like React using JSX or TypeScript and I finally agreed with [Wes Bos](https://wesbos.com/) that VS Code is better for development of React JS. Obviously, with VS Code, you should be having the right extensions to support your React JS development, but comparing with Sublime Text 2, VS Code is still in its infancy. Get these extensions and here's my [dotfile](https://gist.github.com/praveenscience/ebb5439f31774ad2fdc14cb9e7de1fc0):
+- [ ] **[Visual Studio Code](https://code.visualstudio.com/)** is the go-to editor for some complex languages like React using JSX or TypeScript and I finally agreed with [Wes Bos](https://wesbos.com/) that VS Code is better for development of React JS. Obviously, with VS Code, you should be having the right extensions to support your React JS development, but compared with Sublime Text 2, VS Code is still in its infancy. Get these extensions and here's my [dotfile](https://gist.github.com/praveenscience/ebb5439f31774ad2fdc14cb9e7de1fc0):
```
Comment (praveenscience) [inline ReadME.md:41]: Both mean the same.

File: ReadME.md [hunk 6]
```diff
@@ -50,7 +50,7 @@ As this is a React JS application, you need the following to support your develo
   
 ### Installation Instructions
 
-[Create React App](https://github.com/facebookincubator/create-react-app) is a comfortable environment for **learning React**, and is the best way to start building **a new [single-page](https://reactjs.org/docs/glossary.html#single-page-application) application** in React. This project makes use of Create React App. It sets up your development environment so that you can use the latest JavaScript features, provides a nice developer experience, and optimises your app for production. You’ll need to have [Node >= 8.10 and npm >= 5.6](https://nodejs.org/en/) on your machine.
+[Create React App](https://github.com/facebookincubator/create-react-app) is a comfortable environment for **learning React**, and is the best way to start building **a new [single-page](https://reactjs.org/docs/glossary.html#single-page-application) application** in React. This project makes use of Create React App. It sets up your development environment so that you can use the latest JavaScript features, provides a nice developer experience, and optimizes your app for production. You’ll need to have [Node >= 8.10 and npm >= 5.6](https://nodejs.org/en/) on your machine.
```
Comment (praveenscience) [inline ReadME.md:53]: We're not in Amreekah. We're either in India or the United Kingdom.

File: ReadME.md [hunk 7]
```diff
@@ -80,7 +80,7 @@ As this is a React JS application, you need the following to support your develo
    ```bash
    npm start
    ```
-3. Open http://localhost:3000/ on your favourite browser. Let anyone from the mentoring team know, if you are facing any troubles.
+3. Open http://localhost:3000/ on your favourite browser. Let anyone from the mentoring team know if you are facing any troubles.
```
Comment (praveenscience) [inline ReadME.md:83]: I'd go 75% with the comma here.

File: ReadME.md [hunk 8]
```diff
@@ -110,15 +110,15 @@ This project is part of the [GirlScript Summer of Code 2022](https://gssoc.girls
 
 ### Frequently Asked Questions(FAQs) for [GSSoC '22](https://gssoc.girlscript.tech/)
 
-> This project is accepting contributions under [GirlScript Summer of Code 2022](https://gssoc.girlscript.tech/) - three months long Open Source program conducted by [GirlScript Foundation](https://gssoc.girlscript.tech/).
+> This project is accepting contributions under [GirlScript Summer of Code 2022](https://gssoc.girlscript.tech/) - three-month long Open Source program conducted by [GirlScript Foundation](https://gssoc.girlscript.tech/).
```
Comment (praveenscience) [inline ReadME.md:113]: Okay, this makes sense. 😊

File: ReadME.md [hunk 9]
```diff
@@ -110,15 +110,15 @@ This project is part of the [GirlScript Summer of Code 2022](https://gssoc.girls
...
+**1. How to start as a beginner to contribute to the project?**
+*There are many beginner-level issues to begin with, which include documentation, development, and design. Also, feel free to make issues for features as well as for reporting bugs. Participants can claim an issue by commenting that they are interested in it. We are also open to new ideas and suggestions.*
 
 **2. How to communicate with us?**
-*All communication will happen on the Slack channel for this particular projects. Mentors can also be reached through GitHub. Avoid DMs and unnecessary mentions on slack and GitHub both. *Checkout [this](https://github.com/CatsInTech/Rezume/issues) issue for more.**
+*All communication will happen on the Slack channel for this particular project. Mentors can also be reached through GitHub. Avoid DMs and unnecessary mentions on slack and GitHub. *Check out [this](https://github.com/CatsInTech/Rezume/issues) issue for more.**
```
Comment (praveenscience) [inline ReadME.md:121]: These are okay, maybe the last line could replace: `[this](...) issue` with `[these issues](...)`.

**Review by praveenscience — CHANGES_REQUESTED** [review]
I am not sure this one's a valid one... But happy to provide some value. Kindly refrain from these kind of PRs before asking.

**Pull request closed without merging**